documentation/bin/createDocu.groovy

```diff
@@ -396,6 +496,26 @@ class Helper {
         }
         docuRelevantSteps
     }
+
+    static resolveDocuRelevantStages(GroovyScriptEngine gse, File stepsDir) {
```

Comment:
This method has a high level of similarity to `resolveDocuRelevantSteps`. We should avoid having the code duplicted. I would like to suggest to refactor it so that we have one method which can be used for both annotations.

Reply:
I wanted to have a clear separation here since the stages are also considered as steps when it comes to the overall parsing.
By having stages separately the processing is simplified later on.

For sure one could think of adding the logic to the same function but then we would need to call it twice to get the step and stage results or return both steps and stages in one go ...

From my perspective the current way of doing it should be good enough.